### PR TITLE
Update setup.py

### PR DESCRIPTION
**Pull request by edgarriba**
### Description
A few sentences describing the changes proposed in this pull request.

### Status
**Ready/Work in progress/Hold**

### Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or new feature that would cause existing functionality to change)
- [ ] New tests added to cover the changes
- [ ] Docstrings/Documentation updated

### PR Implementer
This is a small checklist for the implementation details of this PR.

If there are any questions regarding code style or other conventions check out our
[summary](https://github.com/kornia/kornia/blob/master/CONTRIBUTING.rst).

- [ ] Did you discuss the functionality or any breaking changes before ?
- [ ] **Pass all tests**: did you test in local ? `make test`
- [ ] Unittests: did you add tests for your new functionality ?
- [ ] Documentations: did you build documentation ? `make build-docs`
- [ ] Implementation: is your code well commented and follow conventions ? `make lint`
- [ ] Docstrings & Typing: has your code documentation and typing ? `make mypy`
- [ ] Update notebooks & documentation if necessary

### KorniaTeam
<details>
  <summary>KorniaTeam workflow</summary>

  - [ ] Assign correct label
  - [ ] Assign PR to a reviewer
  - [ ] Does this PR close an Issue? (add `closes #IssueNumber` at the bottom if
        not already in description)

</details>

### Reviewer
<details>
  <summary>Reviewer workflow</summary>

  - [ ] Do all tests pass? (Unittests, Typing, Linting, Documentation, Environment)
  - [ ] Does the implementation follow `kornia` design conventions?
  - [ ] Is the documentation complete enough ?
  - [ ] Are the tests covering simple and corner cases ?

</details>

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -115,28 +115,45 @@ def run(self):
...
+            'Development Status :: 4 - Beta',
+            # Indicate who your project is intended for
             'Intended Audience :: Developers',
             'Intended Audience :: Education',
             'Intended Audience :: Science/Research',
-            'Operating System :: POSIX :: Linux',
-            'Programming Language :: Python :: 3 :: Only',
-            'License :: OSI Approved :: Apache Software License',
+            'Topic :: Scientific/Engineering :: Artificial Intelligence',
             'Topic :: Scientific/Engineering :: Image Recognition',
```
Comment (shijianjian) [inline setup.py:148]: Is it better to be computer vision than image recognition?

Also, probably similar to the keywords?

```suggestion
            'Topic :: Scientific/Engineering :: Computer Vision',
```

File: setup.py [hunk 2]
```diff
@@ -115,28 +115,45 @@ def run(self):
...
         python_requires='>=3.6',
-        # Test
         setup_requires=['pytest-runner'],
         tests_require=['pytest'],
-        # Package info
         packages=find_packages(exclude=('docs', 'test', 'examples')),
         package_data={"kornia": ["py.typed"]},
         zip_safe=True,
         install_requires=requirements,
+        keywords=['computer vision', 'deep learning', 'pytorch']
```
Comment (shijianjian) [inline setup.py:131]:
```suggestion
        keywords=['computer vision', 'deep learning', 'pytorch'],
```

File: setup.py [hunk 3]
```diff
@@ -115,28 +115,45 @@ def run(self):
...
+        project_urls={
+            "Bug Tracker": "https://github.com/kornia/kornia/issues",
+            "Documentation": "https://kornia.readthedocs.io/en/latest/",
```
Comment (shijianjian) [inline setup.py:134]: Just to be more consistent XD.
```suggestion
            "Documentation": "https://kornia.readthedocs.io/en/latest",
```

File: setup.py (hunk 1 above)
Comment (edgarriba) [reply setup.py:148]: It’s a predefined list. See: https://pypi.org/classifiers/

File: setup.py [hunk 4]
```diff
@@ -115,28 +115,45 @@ def run(self):
...
         zip_safe=True,
         install_requires=requirements,
+        keywords=['computer vision', 'deep learning', 'pytorch'],
+        project_urls={
+            "Bug Tracker": "https://github.com/kornia/kornia/issues",
+            "Documentation": "https://kornia.readthedocs.io/en/latest",
+            "Source Code": "https://github.com/kornia/kornia",
+        },
         classifiers=[
+            'Environment :: Console',
```
Comment (cceyda) [inline setup.py:138]: these tags also look relevant

'Environment :: GPU',
'Intended Audience :: Information Technology',
'Topic :: Software Development :: Libraries',

**Review by cceyda — APPROVED** [review]
these tags also look relevant

'Environment :: GPU',
'Intended Audience :: Information Technology',
'Topic :: Software Development :: Libraries',
